websites/0-9/9anime/presence.ts

```diff
@@ -30,53 +30,56 @@ presence.on("UpdateData", async () => {
 		!isNaN(video.duration) &&
 		document.location.pathname.includes("/watch/")
 	) {
-		const [startTimestamp, endTimestamp] = presence.getTimestamps(
-				Math.floor(video.currentTime),
-				Math.floor(video.duration)
-			),
-			coverArt = document.querySelector<HTMLImageElement>("#w-info img")?.src,
+		delete presenceData.startTimestamp;
+		[, presenceData.endTimestamp] = presence.getTimestamps(
```

Comment:
Don't see why we should remove startTimestamp

Reply:
> Don't see why we should remove startTimestamp
Ok

Reply:
Is this suggestion going to be taken on board or forgotten about?